### ISSUE TITLE
crash when trying to edit a button that also has a shift modifier command

### ISSUE THREAD

**Issue by ntenga** [Ryochan7/DS4Windows#1023]
i can edit all the other buttons just fine, and the buttons with the shift modifier work as intended but if i go back and try to modify them again the modify screen does not open and DS4Windows crashes.
added video

[6Jx7siyywk.zip](https://github.com/Ryochan7/DS4Windows/files/4035110/6Jx7siyywk.zip)

**Comment by Tiberiusmoon** [comment]
[ds4windows_log.txt](https://github.com/Ryochan7/DS4Windows/files/4035201/ds4windows_log.txt)
Confirmed; Crash log added.

Temporary work around: Use the Record a Macro function. . . why not use this only in the first place?

**Comment by ntenga** [comment]
> Temporary work around: Use the Record a Macro function. . . why not only use this in the first place?

sorry if i am missing something, a macro would just run a predetermined series of key presses. what does that have to do with the shift modifier isn't that like the function key you see in some laptops that basically doubles a key's functions?

**Comment by Tiberiusmoon** [comment]
I thought the shift modifier was just holding down the shift key?
If so you can just Press shift as a recorded key press which would do the same thing.
Or any key your able to press on the keyboard like Fn/function.

**Comment by ntenga** [comment]
> 
> 
> I thought the shift modifier was just holding down the shift key?
> If so you can just Press shift as a recorded key press which would do the same thing.
> Or any key your able to press on the keyboard like Fn/function.

yeah you can do that, but isn't what shift modifier does. for example, if you look at my video you can see that the d-pad down has 2 commands in it. down and volume down. 
if i just press d-pad down it uses the command down
if i press d-pad down while i hold the options key ( you can choose which ever key you want, i choose option in this case) it will execute a different command, volume down.

tldr: so using shift modifier one button can do two things. Using macros would use one button to do one thing (albeit complex it is still one thing)

**Comment by Tiberiusmoon** [comment]
Ohh its a input overlay, sorry i just assumed its a hold shift while using a input like a capital letter.
. . . Again the wording of DS4's functions is confusing >.<

**Comment by Ryochan7** [comment]
I will double check that type of binding but editing buttons with Shift Modifiers works fine for me.

Update: Just got a crash there. It happens anytime that a keyboard key is bound in a Shift Modifier. Mapping Unbound or an Xbox 360 button works fine.
